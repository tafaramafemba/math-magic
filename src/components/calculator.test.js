describe('Calculator', () => {
    test('Placeholder', () => {
    expect(1+1).toBe(2)
    })
  })