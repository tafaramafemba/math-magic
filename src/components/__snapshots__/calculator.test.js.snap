// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Calculator Component Rendering  renders correctly 1`] = `
<div
  className="container"
>
  <h2
    className="math"
  >
    Lets do some math!
  </h2>
  <div
    className="calculator"
  >
    <div
      className="input-div"
    >
      <h2
        className="input"
      >
        0
      </h2>
    </div>
    <div
      className="row"
    >
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        AC
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="+/-"
      >
        +/-
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="%"
      >
        %
      </button>
      <button
        className="orange-btn"
        onClick={[Function]}
        type="button"
        value="รท"
      >
        รท
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="7"
      >
        7
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="8"
      >
        8
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="9"
      >
        9
      </button>
      <button
        className="orange-btn"
        onClick={[Function]}
        type="button"
        value="x"
      >
        x
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="4"
      >
        4
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="5"
      >
        5
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="6"
      >
        6
      </button>
      <button
        className="orange-btn"
        onClick={[Function]}
        type="button"
        value="-"
      >
        -
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="1"
      >
        1
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="2"
      >
        2
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="3"
      >
        3
      </button>
      <button
        className="orange-btn"
        onClick={[Function]}
        type="button"
        value="+"
      >
        +
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="grey-btn plus"
        onClick={[Function]}
        type="button"
        value="0"
      >
        0
      </button>
      <button
        className="grey-btn"
        onClick={[Function]}
        type="button"
        value="."
      >
        .
      </button>
      <button
        className="orange-btn"
        onClick={[Function]}
        type="button"
        value="="
      >
        =
      </button>
    </div>
  </div>
</div>
`;
